openff/benchmark/analysis/new_analysis.py

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env python
+
+"""
+new_analysis.py
+
+For 2+ sets of SDF files that are analogous in terms of molecules and their conformers,
+assess them (e.g., having FF geometries) with respective to a reference SDF
+file (e.g., having QM geometries). Metrics include: RMSD of conformers, and relative energy differences.
+
+By:      David F. Hahn, Lorenzo D'Amore
+Version: May 6 2021
+
+"""
+
+import os
+import numpy as np
+import pandas as pd
+import warnings
+
+from rdkit import Chem
+from rdkit.Chem import rdMolAlign
+from tqdm import tqdm
+
+from . import metrics, readwrite
+
+def lucas(input_path, ref_method, output_directory="./5-results-lucas"):
```

Comment:
Can you add a docstring to this function describing what it takes, what it does, etc.? You can use your notes from the PR and sculpt them into shape for here.

Same for the `swope` function.